### PR TITLE
[dashboard] Add small UI changes for desktop IDEs

### LINKED ISSUE #6438: Small UI changes for Desktop IDE configuration dashboard (JetBrains)
A collection of smaller UI suggestions for the JetBrains dashboard.

Changes: 

1. Update the `IdeReady` helper text when IDE is loading from `Your Workspace is Ready` to be: `Running` ([Source](https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734677553))

2. Update `Use Desktop IDE` text on the configuration page to be: `Open in Desktop IDE` ([source](https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734679322))

3. Reorder the primary action button to the right when opening the IDE ([Source](https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734689957))

### PR DESCRIPTION
**Pull request by corneliusludmann**
## Description
This PR adds some smaller UI changes for the desktop IDE feature from issue #6438.

## Related Issue(s)

Fixes #6438

## How to test
Open the settings page and start an IntelliJ workspace. Verify that the items in the issue are fixed.

## Release Notes

```release-note
NONE
```
/no-cc

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **corneliusludmann** and additionally assign **jankoehnlein** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankoehnlein` in a comment when ready.

Associated issue: *#6438*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/settings/Preferences.tsx
Comment (gtsiolis) [inline components/dashboard/src/settings/Preferences.tsx:94]: **thought**: I still think Open VS Code in Browser could be improved but let's leave this out of the scope of this PR for now. Future improvements[[1](https://github.com/gitpod-io/gitpod/issues/6438#issuecomment-961479669)][[2](https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734699665)] could improve make that button or make it redundant.

**Review by gtsiolis — APPROVED** [review]
Flawless! 💯 

Thanks @corneliusludmann for making these changes and @atduarte @loujaybee for participating in the discussion[[1](https://github.com/gitpod-io/gitpod/issues/6438#issuecomment-961717650)][[2](https://github.com/gitpod-io/gitpod/issues/6438#issuecomment-961757072)]! 🏀

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: a132b810e872ab44d31e1d004a29a46ae6f4ca51</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6507#pullrequestreview-799238720" title="Approved">gtsiolis</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **corneliusludmann** after the PR has been reviewed.

Associated issue: *#6438*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
**fyi**: @corneliusludmann this still needs an approval from someone from the ide team. Cc @loujaybee

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6507#issuecomment-962148628" title="Approved">corneliusludmann</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6507#pullrequestreview-799238720" title="Approved">gtsiolis</a>*

Associated issue: *#6438*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [corneliusludmann]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
